app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -395,6 +396,19 @@ def save_container_volumes_inventory(container_group, hashes, target = nil)
     store_ids_for_new_records(container_group.container_volumes, hashes, :name)
   end
 
+  def save_additional_attributes_inventory(entity, hashes, target = nil)
```

Comment:
I agree...the model should just be `has_many :custom_attributes` instead of trying to create some special naming.  If store_ids_for_new_records is needed, there's no reason it can't be added to the existing save_custom_attributes_inventory